unit-tests-java8/src/test/java/org/eclipse/collections/test/CollectionTestCase.java

```diff
@@ -93,7 +94,7 @@ default void Collection_add()
     }
 
     @Test
-    default void Collection_remove()
+    default void Collection_remove_removeAll()
```

Comment:
At first I wrote this as a separate test method. But then I realized the two methods are declared on the same interface, and they were only overridden together and only once, in the unmodifiable subclass.